build.rs

```diff
@@ -38,7 +38,15 @@ static CBL_LIB_DIR: &str = "libcblite-3.0.3/lib";
 fn main() -> Result<(), Box<dyn Error>> {
     generate_bindings()?;
     configure_rustc()?;
-    copy_lib()?;
+
+    // if we're currently in a cargo check workflow, no need to try to copy libs around.
+    // In particular, it won't work using the current clippy_extra on billeo-engine side
```

Comment:
Please no information about billeo-engine, the goal is for this repo to be open source :-)

Reply:
Ok, should I just remove the comment then?

Reply:
You can, or write a simpler one that just points out that copying libs is not necessary in the context of a cargo check.
As you prefer :-)